.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,92 @@
 ---
-name: Bug report
-about: Create a report to help us improve
+name: 🐞 Bug report
+about: Report a bug
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+/kind bug
 
-**To Reproduce**
-Steps to reproduce the behavior:
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+<!--
 
-**Expected behavior**
-A clear and concise description of what you expected to happen.
+Welcome! - We kindly ask you to:
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+  1. Fill out the issue template below 
+  2. Use Gitter if you have a question rather than a bug or feature request.
+    Gitter community channel is at: https://gitter.im/devfile/community
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - Browser [e.g. chrome, safari]
- - Version [e.g. 22]
+Thanks for understanding, and for contributing to the project!
+
+-->
+
+**Which area this bug is related to?**
+
+> Uncomment appropriate `/area` lines, and delete the rest.
+> For example, `> /area api` would simply become: `/area api`
+
+> /area ci
+> /area api
+> /area library
+> /area registry
+> /area devworkspace
+> /area integration-tests
+> /area test-automation
+
+
+## What versions of software are you using?
+
+**Operating System:**
+
+**Go Pkg Version:**
+
+
+## Bug Summary
+**Describe the bug:**
+<!-- 
+    A clear and concise description of what the bug is.
+-->
+
+**To Reproduce:**
+<!--
+    Steps to reproduce the behavior.
+-->
+
+
+## Expected behavior
+<!--
+    A clear and concise description of what you expected to happen.
+-->
+
+## Any logs, error output, screenshots etc? Provide the devfile that sees this bug, if applicable.
+<!--
+To get logs:
+    ci: please copy the github workflow output
+    api: please copy the terminal output
+    library: please copy the terminal output
+    registry: follow instruction under "Collecting Logs" to find log: https://github.com/devfile/registry-support/blob/main/TROUBLESHOOTING.md
+    devworkspace: copy the logs from the controller (kubectl logs deploy/devworkspace-controller -n $NAMESPACE)
+    test-automation: 
+        api: follow instruction under "Running tests locally" to find test log: https://github.com/devfile/api/tree/main/test
+        library: follow instruction under "Running the tests locally" to find test log: https://github.com/devfile/library/tree/main/tests
+        devworkspace: copy the logs from the controller (kubectl logs deploy/devworkspace-controller -n $NAMESPACE)
+    integration-tests: please copy the build log under prow ci result for QE ingetration tests
```

Comment:
Are the "prow ci result" documented somewhere?   We may need to update the readme in the `integration-tests` repo and link to it.  Also, there's a typo on `integration`

Reply:
it's just the integration test result page. If the bug reporter knows it is a bug for `/area integration-test`, they should know how to see the test result. 
Joseph has created an issue for updating integration test's readme to include "collecting logs" section. https://github.com/devfile/api/issues/507
We can update the template to refer to that readme after that issue has been closed.

Reply:
updated